src/NuGet.Clients/PackageManagement.PowerShellCmdlets/Cmdlets/UpdatePackageCommand.cs

```diff
@@ -211,33 +214,51 @@ private void PerformPackageUpdatesOrReinstalls()
                     Token);
             }
 
-            // set _installed to true, if package to update is installed.
-            _isPackageInstalled = actions.Any();
-
             await ExecuteActions(actions);
+
+            return isInstalled;
+        }
+
+        /// <summary>
+        /// Method checks if the package to be updated is installed in any package or not.
+        /// </summary>
+        /// <param name="packageId">Id of the package to be updated/checked</param>
+        /// <returns><code>bool</code> indicating wether the package is installed on any package or not</returns>
+        private async Task<bool> IsPackageInstalled(string packageId)
+        {
+            foreach (var project in Projects)
+            {
+                var installedPackages = await project.GetInstalledPackagesAsync(CancellationToken.None);
```

Comment:
Shouldn't this use [Token](https://github.com/NuGet/NuGet.Client/blob/5741b065cb5c27a9e82f3c67056f46cc80ab550c/src/NuGet.Clients/PackageManagement.PowerShellCmdlets/NuGetPowerShellBaseCommand.cs#L155), a property inherited from the base class `NuGetPowerShellBaseCommand`?

Side note:  the superclass `PackageActionBaseCommand` also has several locations which use `CancellationToken.None`.  Not sure how many of those should also use `Token`.